src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -2015,8 +2028,21 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       <Output TaskParameter="FilesWritten" ItemName="FileWrites"/>
       <Output TaskParameter="DependsOnSystemRuntime" PropertyName="DependsOnSystemRuntime"/>
     </ResolveAssemblyReference>
+
+    <ItemGroup>
+      <ReferencePathWithInterfaceOnlyAssemblies Include="@(ReferencePath->'%(ReferenceAssembly)')" />
```

Comment:
Is it OK to have empty strings for the ReferencaPath items that do not have ref assemblies?